README.md

```diff
@@ -100,24 +100,7 @@ Ruff also integrates with [`coc.nvim`](https://github.com/neoclide/coc.nvim/wiki
 ### Example: Sublime Text
 
 To use `ruff-lsp` with Sublime Text, install Sublime Text's [LSP](https://github.com/sublimelsp/LSP)
-package, then add something like the following to `LSP.sublime-settings`:
-
-```json
-{
-  "clients": {
-    "ruff-lsp": {
-      "command": ["ruff-lsp"],
-      "enabled": true,
-      "selector": "source.python",
-      "initializationOptions": {
-        "settings": {
-          "args": []
-        }
-      }
-    }
-  }
-}
-```
+and [LSP-ruff](https://github.com/sublimelsp/LSP-ruff) package.
```

Comment:
Amazing! Can we include an example of how you would configure it, e.g., to change one of the settings?

Reply:
Should I add that to the LSP-ruff Readme instead? This way people coming from package control (the ST package manager) can profit from that too.

Reply:
I added a section to the LSP-ruff readme.

Reply:
In Sublime Text, the settings are displayed as a split view:
![image](https://user-images.githubusercontent.com/8215084/219061392-505026c8-33c5-4fa5-9234-a8602bebcabc.png)

Reply:
Thanks!